### PR TITLE
Tapping on News image should load page #648

### PR DESCRIPTION
**Pull request by aleksandr-cliqz**
Fix #
https://facebook.github.io/react-native/docs/touchablewithoutfeedback

## Checklist:

- [ ] My squashed commit messages follows the [7 golden rules](https://chris.beams.io/posts/git-commit/)
- [ ] I updated or created necessary unit tests
- [ ] I've tested my changes in Dark Mode, iPad, iOS <= 12, Landscape and in German

### CODE REVIEW

File: ReactNative/js/screens/Home/components/News.jsx [hunk 1]
```diff
@@ -57,15 +57,16 @@ const useNews = newsModule => {
...
+  const { style, source } = props;
+  const [isHidden, setHidden] = useState(false, [source]);
   const hide = useCallback(() => setHidden(true), [setHidden]);
   const hiddenStyle = useMemo(
     () => (isHidden ? { height: 0, marginBottom: 0 } : null),
     [isHidden],
   );
   return (
-    <Image style={[style, hiddenStyle]} source={{ uri: url }} onError={hide} />
+    <Image {...props} style={[style, hiddenStyle]} source={{ uri: source }} onError={hide} />
```
Comment (chrmod) [inline ReactNative/js/screens/Home/components/News.jsx]:
```suggestion
    <Image style={[style, hiddenStyle]} source={{ uri: source }} onError={hide} />
```
no need to spread the props as all props are already passed

Comment (aleksandr-cliqz) [reply ReactNative/js/screens/Home/components/News.jsx]: ```TouchableWithoutFeedback``` provide implicit props which should be pass
From documentation 
```
function MyComponent(props) {
  return (
    <View {...props} style={{flex: 1, backgroundColor: '#fff'}}>
      <Text>My Component</Text>
    </View>
  );
}

<TouchableWithoutFeedback onPress={() => alert('Pressed!')}>
  <MyComponent />
</TouchableWithoutFeedback>;
```
note plz how they spread props 
https://facebook.github.io/react-native/docs/touchablewithoutfeedback

**Comment by aleksandr-cliqz** [comment]
@chrmod all interesting seems ```implicit``` :)

**Comment by aleksandr-cliqz** [comment]
@chrmod could you take a look again plz?

**Pull request merged**